package.json

```diff
@@ -74,8 +74,8 @@
     "vitest": "^0.7.0"
   },
   "peerDependencies": {
-    "react": "^16.9.0 || ^17",
-    "react-dom": "^16.9.0 || ^17"
+    "react": "^18",
```

Comment:
Is it ok to set something like `^16.9.0 || ^17 || ^18` for the backward compatibility?

Reply:
I think you are right. There is nothing wrong with the source code so there was no need to remove support.
I have modified the code as you pointed out.